src/display.rs

```diff
@@ -423,16 +435,43 @@ impl Display {
                     // Iterate over all non-empty cells in the grid
                     for cell in grid_cells {
                         // Update underline/strikeout
-                        cell_line_rects.update_lines(&cell);
+                        rects.update_lines(&cell);
 
                         // Draw the cell
                         api.render_cell(cell, glyph_cache);
                     }
                 });
             }
 
-            // Draw rectangles
-            self.renderer.draw_rects(config, &size_info, visual_bell_intensity, cell_line_rects);
+            if let Some(message) = bar_message {
+                // Create a new rectangle for the background
+                let text = message.text(&size_info);
+
+                let start_line = (size_info.lines().0 - text.len()) as f32;
+                let y = size_info.padding_y + size_info.cell_height * start_line;
```

Comment:
It's much easier to say no now, than to refactor a bunch of crap later. The more nitpicky the better!

I don't think this is a super big issue, I originally moved it from the `let y = ...` line up there because of line length limitations. Both `usize` and `f32` aren't the `Line` type, so one could argue that one of them isn't any better than the other. I've decided to still move it though because it actually removes a pair of parentheses.